### PR TITLE
Doc/fix

### PR DESCRIPTION
**Pull request by chrpetre**
fix few errors in documentation
update licensing requirements section
update doc/image of example with input fields
update doc dependencies for pyfluent

### CODE REVIEW

File: examples/evaluate/06-TBROM_input_field.py [hunk 1]
```diff
@@ -1,40 +1,42 @@
...
    To be able to use the functionalities to project an input field snapshot, you must have a
    twin with one or more TBROMs parameterized by input field data. Input mode coefficients
    for TBROMs are connected to the twin's inputs following these conventions:
-   - If there are multiple TBROMs in the twin, the format for the name of the
-      twin input is ``"input field name"_mode_"i"_"tbrom name"``.
-   - If there is a single TBROM in the twin, the format for the name of the twin
-      input is ``"input field name"_mode_"i"``.
+
+   - If there are multiple TBROMs in the twin, the format for the name of the twin input must be
+     ``"input field name"_mode_"i"_"tbrom name"``.
```
Comment (lboucin) [inline examples/evaluate/06-TBROM_input_field.py:16]: (Maybe more pythonic?) suggestion: f"{input_field_name}_mode_{mode_index}_{tbrom_name}"

File: examples/evaluate/06-TBROM_input_field.py [hunk 2]
```diff
@@ -1,40 +1,42 @@
...
+   - If there is a single TBROM in the twin, the format for the name of the twin input must be
+     ``"input field name"_mode_"i"``.
```
Comment (lboucin) [inline examples/evaluate/06-TBROM_input_field.py:18]: (Maybe more pythonic?) suggestion: f"{input_field_name}_mode_{mode_index}"

File: examples/evaluate/06-TBROM_input_field.py [hunk 3]
```diff
@@ -1,40 +1,42 @@
...
 
    To be able to use the functionalities to generate an output field snapshot on demand, you
    must have a twin with one or more TBROMs. The output mode coefficients for the TBROMs
    must be enabled when exporting the TBROMs and connected to twin outputs with following
    these conventions:
+
    - If there are multiple TBROMs in the twin, the format for the name of the twin
-     output is ``"outField_mode_"i"_"tbrom name"``.
+     output must be ``"outField_mode_"i"_"tbrom name"``.
```
Comment (lboucin) [inline examples/evaluate/06-TBROM_input_field.py:26]: (Maybe more pythonic?) suggestion: f"outField_mode_{mode_index}_{tbrom_name}"

File: examples/evaluate/06-TBROM_input_field.py [hunk 4]
```diff
@@ -1,40 +1,42 @@
...
    - If there is a single TBROM in the twin, the format for the name of the twin
-     output is ``"outField_mode_"i"``.
+     output must be ``"outField_mode_"i"``.
```
Comment (lboucin) [inline examples/evaluate/06-TBROM_input_field.py:28]: (Maybe more pythonic?) suggestion: f"outField_mode_{mode_index}"

**Review by lboucin — APPROVED** [review]
Made minor docstring suggestions. LGTM otherwise!
Thanks

File: examples/evaluate/06-TBROM_input_field.py (hunk 1 above)
Comment (chrpetre) [reply examples/evaluate/06-TBROM_input_field.py]: applied change

File: examples/evaluate/06-TBROM_input_field.py (hunk 3 above)
Comment (chrpetre) [reply examples/evaluate/06-TBROM_input_field.py]: applied change

**Pull request merged**